src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+* {
```

Comment:
it's better not to use `*` selector. better to add all specific selectors to reset default styles.
you can find more info here: https://meyerweb.com/eric/tools/css/reset/